contentcuration/contentcuration/frontend/channelEdit/pages/StagingTreePage/index.vue

```diff
@@ -423,23 +423,33 @@
       },
     },
     created() {
-      if (!this.hasStagingTree) {
-        return;
-      }
-
       this.isLoading = true;
-      Promise.all([
-        this.loadAncestors({ id: this.nodeId, includeSelf: true }),
-        this.loadChildren({ parent: this.nodeId, tree_id: this.stagingId }),
-      ]).then(() => (this.isLoading = false));
-
+      this.$store
+        .dispatch('currentChannel/loadChannel', { staging: true })
+        .then(channel => {
+          if (channel.staging_root_id) {
+            return this.loadTree({ tree_id: channel.staging_root_id });
+          }
+        })
+        .catch(error => {
+          throw new Error(error);
+        })
+        .finally(() => {
+          if (!this.hasStagingTree) {
+            return;
+          }
+          Promise.all([
+            this.loadAncestors({ id: this.nodeId, includeSelf: true }),
+            this.loadChildren({ parent: this.nodeId, tree_id: this.stagingId }),
+          ]).then(() => (this.isLoading = false));
+        });
       this.loadCurrentChannelStagingDiff();
     },
     methods: {
       ...mapActions(['showSnackbar', 'addViewModeOverride', 'removeViewModeOverride']),
       ...mapActions('channel', ['loadChannel']),
       ...mapActions('currentChannel', ['loadCurrentChannelStagingDiff', 'deployCurrentChannel']),
```

Comment:
There are already `currentChannel` actions mapped here so it would be nice to add `loadChannel` to them instead of calling `'currentChannel/loadChannel'